admin/console/hidden-settings.rst

```diff
@@ -96,13 +113,16 @@ Get the current state of this setting with:
 
    >> Setting.get('ui_user_organization_selector_with_email')
 
-
 Change Font-Settings for outgoing HTML-Mails
 --------------------------------------------
 
-.. note:: Some Clients (like Outlook) might fallback to other Settings while it might work for other Clients.
+.. note::
+
+   Some Clients (like Outlook) might fallback to other Settings while it might
+   work for other Clients.
```

Comment:
"Clients" (twice) and "Settings" are no proper nouns, therefore should not be capitalized.